src/pages/404.tsx

```diff
@@ -47,7 +47,7 @@ const NotFoundPage = (): JSX.Element => {
   );
 };
 
-export const getStaticProps: GetStaticProps = async ({ locale }) => {
+export const getStaticProps: GetStaticProps = async ({ locale = 'en' }) => {
```

Comment:
```suggestion
export const getStaticProps: GetStaticProps = async ({ locale }) => {
```
this is automatically added by next-i18next in https://github.com/MenheraBot/MenheraSite/blob/NewSite/next-i18next.config.js#L6

Reply:
@ySnoopyDogy 
suggest it

Reply:
This not working:

![image](https://user-images.githubusercontent.com/66279500/136114290-f92e5aeb-364d-42d1-9a17-5f0f955e7426.png)

My code does not set the locale to en the locale is en only in case it is null.
It's like that only less verbose

```tsx
const locale = locale ? locale : 'en'
```

![image](https://user-images.githubusercontent.com/66279500/136114322-e768e590-f9e6-4773-9337-d26ddcdd3c5a.png)